packages/react-vanilla-components/src/components/ResponsiveGrid.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useCallback, useContext } from 'react';
+import { FormContext, getRenderer } from '@aemforms/af-react-renderer';
+import { PROPS_PANEL } from '../utils/type';
+
+
+const ResponsiveGrid = (props: PROPS_PANEL) => {
+    //   @ts-ignore
+    const context = useContext(FormContext);
+
+    const getItemClassNames = (appliedCssClassNames: string, index: number, fieldType: string) => {
```

Comment:
update this method 
const getItemClassNames = (defaultClass: string, index: number, fieldType: string) => {
    let classNames =  defaultClass;
    if(props[':itemsOrder'] && props.columnClassNames){
        classNames += ` ${props.columnClassNames[props[':itemsOrder'][index]]} ${fieldType === 'panel' ? ' responsivegrid' : ' base'}`
    }
    return classNames;
};

Reply:
implemented!